optaplanner-core/src/main/java/org/optaplanner/core/impl/phase/AbstractPhase.java

```diff
@@ -107,14 +107,12 @@ public void setAssertShadowVariablesAreNotStaleAfterStep(boolean assertShadowVar
     public void solvingStarted(DefaultSolverScope<Solution_> solverScope) {
         // bestSolutionRecaller.solvingStarted(...) is called by DefaultSolver
         termination.solvingStarted(solverScope);
-        phaseLifecycleSupport.fireSolvingStarted(solverScope);
```

Comment:
Very nice bug finding.

I 'll merge this PR after the big local refactor I am working on for initScore is merged (and I 'll deal with any merge conflicts).